docs/hello_world.rst

```diff
@@ -0,0 +1,228 @@
+The purpose of this document is to showcase creation of a basic Gumby experiment.
+This document assumes the reader has a basic understanding of creating Dispersy communities.
```

Comment:
Maybe add a reference to the dispersy documentation here? dispersy.readthedocs.org